baseplate.go

```diff
@@ -235,10 +235,11 @@ func DecodeConfigYAML(reader io.ReadSeeker, serviceCfg interface{}) (Config, err
 
 // NewArgs defines the args used in New functino.
 type NewArgs struct {
-	EdgeContextImpl ecinterface.Interface
-
 	ConfigPath string
 
+	// The factory used to create edge context implementation.
```

Comment:
We should probably say that this is required and `baseplate.New` will panic if it is not set?